### PR TITLE
fix(values): update value names for correct usage

### PR DESCRIPTION
**Pull request by mindreeper2420**
Update the value names to reference proper values within the system.

@aslakknutsen 
Can you verify if this PR fixes both https://github.com/fabric8-services/fabric8-notification/issues/12 and https://github.com/fabric8-services/fabric8-notification/issues/15?

### CODE REVIEW

**Comment by aslakknutsen** [comment]
Are we sure we want to use title in all of these locations? Title can get fairly long:

A comment has been added to 1343 UX - Workflow for type groups (portfolio and requirements) navigation and behavior in the side panel.

**Comment by mindreeper2420** [comment]
@aslakknutsen I truncated the titles, so it will resemble what the Detail Panel shows now.
screen shot 2017-07-24 at 9 12 10 am

File: template/work-item-comment-created/work-item-comment-created.html [hunk 1]
```diff
@@ -31,7 +35,7 @@
           </td>
           <td class="table-content" style="text-align: left;">
             <a href="#" style="color: #0088ce; text-decoration: none;">
-              <img src="profile_thumbnail.png" />Adam J. Jolicoeur
```
Comment (aslakknutsen) [inline template/work-item-comment-created/work-item-comment-created.html]: No more gravatar icon, or?

Comment (aslakknutsen) [inline template/work-item-comment-created/work-item-comment-created.html]: Don't have to fix it in the templates. I've moved them over to a slightly different format

**Pull request merged**